tools/deployment/osquery.example.conf

```diff
@@ -59,7 +63,17 @@
     }
   },
 
-  /* Add default osquery packs or install your own. */
+  /**
+   * Add default osquery packs or install your own.
+   *
+   * There are several 'default' packs installed with 'make install' or via
+   * packages and/or Homebrew.
+   *
+   * Linux:        /usr/share/osquery/packs
+   * OS X:         /var/osquery/packs
+   * Homebrew:     /usr/local/share/osquery/packs
+   * make install: {PREFIX}/share/osquery/packs
+   */
```

Comment:
My thoughts were: for sections use C-style comments, and otherwise C++ (for longer field explanations and to disable text). There's also an auxiliary benefit of demoing the use of both. ;)